app/Models/AuthorRepository.php

```
@@ -468,6 +487,74 @@ public function mergeAuthors(array $authorIds, ?int $primaryId = null): ?int
                 }
                 $stmt->close();
 
+                // Repoint authority alternates before the cascade drops them.
+                // Surrogate PK only (no unique on autore_id + authority), so a
+                // plain UPDATE cannot collide.
+                $stmt = $this->db->prepare(
+                    "UPDATE author_authority_alternates SET autore_id = ? WHERE autore_id = ?"
+                );
+                if ($stmt === false) {
+                    throw new \Exception("Failed to prepare UPDATE author_authority_alternates: " . $this->db->error);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '497', column '31'). (undefined)

(MissingImport)